engine-rest/engine-rest-openapi/src/main/templates/models/org/camunda/bpm/engine/rest/dto/history/HistoricJobLogQueryDto.ftl

```diff
@@ -0,0 +1,20 @@
+<#-- Generated From File: camunda-docs-manual/public/reference/rest/history/job-log/post-job-log-query/index.html -->
+<#macro dto_macro docsUrl="">
+<@lib.dto desc = "A Historic Job Log instance query which defines a list of Historic Job Log instances">
+    
+<#assign requestMethod="POST">
```

Comment:
I cannot see that we use the request method in the `history-job-log-params.ftl`. Should we remove it here? It also does no harm as far as I can tell but I guess it's also not necessary?

Reply:
You are right, it is not necessary but it also does not hurt. I'll remove it, non the less.